src/payments/services/premium-certificate-purchase-email.service.ts

```
@@ -0,0 +1,165 @@
+    if (nested != null && typeof nested === 'object') {
+      const inner = extractDownloadUrlFromCertificateResponse(nested);
+      if (inner) {
+        return inner;
+      }
+    }
+  }
+  return '';
+}
+
+function formatPaidAtIst(date: Date): string {
+  return new Intl.DateTimeFormat('en-US', {
```

ESLint

[error] Replace `'en-US'` with `"en-US"`

(prettier/prettier)